frontend/components/organisms/AnotherUserProfileModal.tsx

```diff
@@ -0,0 +1,222 @@
+                  >
+                    <GoGoal size="20px" color="green" />
+                  </Box>
+                )}
+
+                {anotherUserProfile?.github ? (
+                  <Flex mt={10}>
+                    <Box
+                      display="flex"
+                      alignItems="center"
+                      justifyContent="center"
+                      borderRadius="full"
+                      borderColor="gray.400"
+                      w="40px"
+                      h="40px"
+                      boxShadow="0px 0px 10px rgba(0, 0, 0, 0.25)"
+                    >
+                      <Icon as={FaGithub} size="20px" color="gray" />
+                    </Box>
+                    <Box ml="20px" fontSize="lg" lineHeight="40px">
+                      <Link href={`${anotherUserProfile?.github}`} isExternal>
+                        <Text>{anotherUserProfile?.github}</Text>
+                      </Link>
+                    </Box>
+                  </Flex>
+                ) : (
+                  <Box
+                    mt={5}
+                    display="flex"
+                    alignItems="center"
+                    justifyContent="center"
+                    borderRadius="full"
+                    borderColor="gray.400"
+                    w="40px"
+                    h="40px"
+                    boxShadow="0px 0px 10px rgba(0, 0, 0, 0.25)"
+                  >
+                    <Icon as={FaGithub} size="20px" color="gray" />
+                  </Box>
+                )}
+
+                {anotherUserProfile?.x ? (
+                  <Flex mt={10}>
+                    <Box
+                      display="flex"
+                      alignItems="center"
+                      justifyContent="center"
+                      borderRadius="full"
+                      borderColor="gray.400"
+                      w="40px"
+                      h="40px"
+                      boxShadow="0px 0px 10px rgba(0, 0, 0, 0.25)"
+                    >
+                      <Icon as={FaXTwitter} size="20px" color="gray" />
+                    </Box>
+                    <Box ml="20px" fontSize="lg" lineHeight="40px">
+                      <Link href={`${anotherUserProfile?.x}`} isExternal>
+                        <Text>{anotherUserProfile?.x}</Text>
+                      </Link>
+                    </Box>
+                  </Flex>
+                ) : (
+                  <Box
+                    mt={5}
+                    display="flex"
+                    alignItems="center"
+                    justifyContent="center"
+                    borderRadius="full"
+                    borderColor="gray.400"
+                    w="40px"
+                    h="40px"
+                    boxShadow="0px 0px 10px rgba(0, 0, 0, 0.25)"
+                  >
+                    <Icon as={FaXTwitter} size="20px" color="gray" />
+                  </Box>
+                )}
```

Comment:
URL登録されてなかったらアイコン出さなくて良いので

```suggestion
                {anotherUserProfile?.x && (
                  <Flex mt={10}>
                    <Box
                      display="flex"
                      alignItems="center"
                      justifyContent="center"
                      borderRadius="full"
                      borderColor="gray.400"
                      w="40px"
                      h="40px"
                      boxShadow="0px 0px 10px rgba(0, 0, 0, 0.25)"
                    >
                      <Icon as={FaXTwitter} size="20px" color="gray" />
                    </Box>
                    <Box ml="20px" fontSize="lg" lineHeight="40px">
                      <Link href={`${anotherUserProfile?.x}`} isExternal>
                        <Text>{anotherUserProfile?.x}</Text>
                      </Link>
                    </Box>
                  </Flex>
                ) 
```

こうしますかね！

Reply:
URLに関してはアイコン省略する様にしました。
ひとことと目標はアイコン常時表示にしました。→データ有無でText部分のみ省略可能にしました